release/attribution.toml

```diff
@@ -0,0 +1,34 @@
+accepted = [
+    "Apache-2.0",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "ISC",
+    "MIT",
+    "OpenSSL",
+    "Unicode-DFS-2016",
+    "Zlib",
+]
+
+targets = [
+    "x86_64-unknown-linux-gnu",
+    "aarch64-unknown-linux-gnu",
+]
+
+ignore-build-dependencies = true
+ignore-dev-dependencies = true
+
+no-clearly-defined = true
+
+workarounds = ["ring"]
+
+[zerocopy.clarify]
+license = "BSD-2-Clause"
+[[zerocopy.clarify.files]]
+path = "LICENSE"
+checksum = "83c1763356e822adde0a2cae748d938a73fdc263849ccff6b27776dff213bd32"
+
+[zerocopy-derive.clarify]
+license = "BSD-2-Clause"
+[[zerocopy-derive.clarify.files]]
+path = "LICENSE"
+checksum = "83c1763356e822adde0a2cae748d938a73fdc263849ccff6b27776dff213bd32"
```

Comment:
Not sure! Without this, cargo-about complains that it can't find the license. cargo-deny seems a little better at finding the license file:
```
help[accepted]: license requirements satisfied
  ┌─ zerocopy 0.6.1 (registry+https://github.com/rust-lang/crates.io-index):5:15
  │
4 │ license = ""
  │            - license expression was not specified
5 │ files-expr = "BSD-2-Clause"
  │               ^^^^^^^^^^^^
  │               │
  │               license expression retrieved via LICENSE
  │               accepted: license is explicitly allowed
  │
  = zerocopy v0.6.1
    └── fuser v0.12.0
        └── mountpoint-s3 v0.3.0

help[accepted]: license requirements satisfied
  ┌─ zerocopy-derive 0.3.2 (registry+https://github.com/rust-lang/crates.io-index):5:15
  │
4 │ license = ""
  │            - license expression was not specified
5 │ files-expr = "BSD-2-Clause"
  │               ^^^^^^^^^^^^
  │               │
  │               license expression retrieved via LICENSE
  │               accepted: license is explicitly allowed
  │
  = zerocopy-derive v0.3.2
    └── zerocopy v0.6.1
        └── fuser v0.12.0
            └── mountpoint-s3 v0.3.0
```

Reply:
Weird. The two projects are both owned by the same org.

Ah well, not an issue.